components/quizSingle/ScoreGraph.tsx

```diff
@@ -1,8 +1,11 @@
-import { GraphSVG, GraphPathBG, GraphPath, GraphText } from "./styles";
```

Comment:
Why are these moved out of /styles?

The point here is to have a seperation of concerns. We should try avoiding having too much CSS in our functional .tsx styles. This is why we have the seperated styles file.